cloud-aws/src/main/java/com/sequenceiq/cloudbreak/cloud/aws/view/AwsCredentialView.java

```diff
@@ -33,11 +53,43 @@ public Boolean isGovernmentCloudEnabled() {
     }
 
     public String getAccessKey() {
-        return cloudCredential.getParameter("accessKey", String.class);
+        String accessKey = cloudCredential.getStringParameter("accessKey");
```

Comment:
Using a `hasParameter` method in `CloudCredential` would be nicer, but I'll fix this.

Reply:
CloudCredentil has not hasParameters method

Reply:
Don't worry about it, I'll add it.